portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,9 +73,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             return;
         }
         for(int i = 0; i < numberOfCommentsToDisplay; i++){
+
             limitedMessages.add(messages.get(i));
         }
+
         response.getWriter().println(gson.toJson(limitedMessages));
+
```

Comment:
Nit: and also remove trailing empty line at the end of a scope?

Reply:
Resolved